spec/factories/uploaded_files.rb

```diff
@@ -0,0 +1,36 @@
+FactoryGirl.define do
+  factory :uploaded_file do
+    title "some title"
+    description "some description"
+
+    membership_application
+
+    FIXTURE_DIR = File.join("#{Rails.root}",'spec','fixtures','uploaded_files')
+    trait :png do
+      actual_file { File.new(File.join(FIXTURE_DIR, 'image.png')) }
+    end
+    trait :gif do
+      actual_file { File.new(File.join(FIXTURE_DIR, 'image.gif')) }
+    end
+    trait :jpg do
+      actual_file { File.new(File.join(FIXTURE_DIR, 'image.jpg')) }
+    end
+
+    trait :pdf do
+      actual_file { File.new(File.join(FIXTURE_DIR, 'diploma.pdf')) }
+    end
+
+    trait :txt do
+      actual_file { File.new(File.join(FIXTURE_DIR, 'specifications.txt')) }
+    end
+
+    trait :exe do
+      actual_file { File.new(File.join(FIXTURE_DIR, 'tred.exe')) }
+    end
+    trait :bin do
+      actual_file { File.new(File.join(FIXTURE_DIR, 'tred.bin')) }
+    end
+
+  end
```

Comment:
Missing .doc and .docx file types.  (I assume these types are supported?  - if not, please ignore)